app/helpers/application_helper.rb

```diff
@@ -22,7 +22,22 @@ def forex_link(price)
     return link_to "(convert)",
       link,
       target: "_blank"
-  end 
+  end
+
+  def build_alert_classes(alert_type)
+    classes = 'alert alert-dismissable '
+    case alert_type.to_sym
+      when :alert, :danger, :error, :validation_errors
+        classes += 'alert-danger'
+      when :warning, :todo
+        classes += 'alert-warning'
+      when :notice, :success
+        classes += 'alert-success'
+      else
+        classes += 'alert-info'
```

Comment:
Should we instead raise an unknown alert type error; and encourage people to explicitly choose that its a .info; or put it a smart default ```build_alert_classes(alert_type = :info)```